ES-services-index-template.json

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
What should I do with this file? Could you append some instructions to this section of the readme? https://github.com/osu-mist/locations-api#post-to-elasticsearch